tests/js/server/all-server-tests.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+var sequence = require("when/sequence");
+var tests = [
+    "./dispatcher-and-inline-tests.js",
+    // TODO:  I cannot get this to play nicely with the promise-based Zombie tests.  Discuss with Antranig.
```

Comment:
When some tests are run in combination, I get seemingly unrelated errors that do not occur when running them singly.  If any of them persist after shooting Zombie in the head, I will bring them up then.